docs/core/man/include/doveadm-backup-sync.inc

```diff
@@ -200,9 +200,13 @@ parameters.
 
 **-t** *start date*
 :   Skip any mails whose received-timestamp is older than the specified time.
+    The format follows IMAP4rev1 standard format, e.g. "13-Apr-2007 14:23:05
+    +0200"
```

Comment:
This is only partially true. Better would be to add a reference to the full syntax possibilities: `For the timestamp format, see the DATE SPECIFICATION in [[man,doveadm-search-query,,7]].`